physics/SFC_Models/Land/Noahmp/noahmpdrv.F90

```diff
@@ -297,6 +299,8 @@ subroutine noahmpdrv_run                                       &
   real(kind=kind_phys), dimension(:)     , intent(in)    :: snow_mp    ! microphysics snow [mm]
   real(kind=kind_phys), dimension(:)     , intent(in)    :: graupel_mp ! microphysics graupel [mm]
   real(kind=kind_phys), dimension(:)     , intent(in)    :: ice_mp     ! microphysics ice/hail [mm]
+  real(kind=kind_phys), dimension(:,:)   , intent(in)    :: mntvar     ! subgrid orographic statistics data
+  real(kind=kind_phys)                   , intent(in)    :: psl_gwd_z0m_factor  ! psl tofd zom factor
```

Comment:
can you be more explicit on this description? psl is unnecessary in the variable name, eventually we want to move to more descriptive variable names, so I need to know what this is, spell out tofd and zom, so this comment should be something like "turbulent orographic form drag momentum roughness length factor"